web/src/index.js

```diff
@@ -124,6 +124,12 @@ if (isMobile) {
     mobileInput.focus();
 } else {
     mobileInput.hidden = true;
+  $(document).keydown((evt) => {
```

Comment:
While this fix seems to work, it doesn't seem to integrate well with what's there: the other event handlers don't use JQuery and there is another keydown handler immediately after this one (and a couple more before).

I suspect that this change is working because the keydown event for the document takes priority over the keydown event for the window, and because the new handler returns false, we never execute the window handler any longer. Which is a different kind of bug.

What about https://github.com/endbasic/endbasic/pull/147 ? Seems simpler and more complete.

Reply:
I used jquery because I got lazy 😅 sorry, first time contributing to a project. The reason I did it this way was because I was unable to get it to register presses while canceling the event earlier - - maybe I made a mistake. Additionally, I'm not sure if that PR works. Returning false from a vanilla JS event handler doesn't do anything, you have to call event.preventDefault() I believe.

Reply:
I don't know what I'm doing either on the JS front 😅, but I tried your change and it seemed to have the right effect, and mine seems to do it as well (though I haven't had a chance to test iOS nor Android yet).

I did some research as well because there wasn't any mention of return values from the event handlers in the jQuery documentation, so I was a bit confused. But I did find a page talking about this and the event.preventDefault() thingy you mention now so... I'm still confused. Need to do some more research to fully understand what's going on.

Reply:
Did some more research. "return false" isn't documented as a way to do this, and seems to be a relic of the past. I've changed #147 to use addEventListener per the doc recommendations and use preventDefault().